src/cli/commands/object/get.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const Command = require('ronin').Command
+const utils = require('../../utils')
+const bs58 = require('bs58')
+const debug = require('debug')
+const log = debug('cli:object')
+log.error = debug('cli:object:error')
+
+module.exports = Command.extend({
+  desc: 'Get and serialize the DAG node named by <key>',
+
+  options: {},
+
+  run: (key) => {
+    var ipfs = utils.getIPFS()
+
+    if (!key) {
+      throw new Error("Argument 'key' is required")
+    }
+
+    if (utils.isDaemonOn()) {
+      return ipfs.object.get(key, (err, obj) => {
+        if (err) {
+          log.error(err)
+          throw err
+        }
+
+        console.log(JSON.stringify(obj))
+      })
+    }
+
+    const mh = new Buffer(bs58.decode(key))
+    ipfs.object.get(mh, (err, obj) => {
+      if (err) {
+        log.error(err)
+        throw err
+      }
+
+      console.log(JSON.stringify({
+        Links: obj.links.map((link) => ({
+          Name: link.name,
+          Hash: bs58.encode(link.hash).toString(),
+          Size: link.size
+        })),
+        Data: obj.data.toString()
```

Comment:
That's because core returns an object with lowercase keys and the multihashes as a buffer, on the other hand the `js-ipfs-api` returns the http API response which returns the object like `go-ipfs`